plugin.js

```diff
@@ -71,11 +72,13 @@ const globals = new Set([
   '_makeShareableClone',
   '_updateDataSynchronously',
   'eval',
+  'evalWithSourceMap',
```

Comment:
I think we no longer need either because of the recent global object fix we can now use global.evalBlah and it doesn't need to be included on the whitelist